WORKSPACE

```diff
@@ -12,9 +12,21 @@ http_archive(
 )
 
 http_archive(
-    name = "io_bazel_rules_docker",
```

Comment:
Speaking of bazel versions. I think you fix the blocker for bazel 7 here :)
https://github.com/buildbarn/bb-storage/issues/185

I never got around to verifying `rules_docker` builds on bazel 7 so thankful for this uplift.
We can look at bazel 7 after this PR :+1:

Reply:
Yes that sounds correct.